PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -2082,6 +2082,7 @@ intptr_t CALLBACK FindReplaceDlg::run_dlgProc(UINT message, WPARAM wParam, LPARA
 
 				case IDC_COPY_MARKED_TEXT:
 				{
+					if (isMacroRecording) saveInMacro(wParam, FR_OP_FIND);
 					(*_ppEditView)->markedTextToClipboard(SCE_UNIVERSAL_FOUND_STYLE);

```

Comment:
Maybe in between the braces for IDC_COPY_MARKED_TEXT, do _only_:  `::SendMessage(_hParent, WM_COMMAND, IDM_SEARCH_MARKEDTOCLIP, 0);`

Reply:
Smart one!